components/ide/code-desktop/status/main.go

```diff
@@ -63,10 +63,15 @@ func main() {
 		}
 		queryString := string(b)
 
+		workspaceLocation := wsInfo.GetWorkspaceLocationFile()
```

Comment:
I think we should align with https://github.com/gitpod-io/openvscode-server/blob/57d3840d9d89487462a2004551d58563022a7324/extensions/gitpod-shared/src/features.ts#L405

workspace folder can be also empty then we should fall back to checkout location

Reply:
fallback is already handled [here](https://github.com/gitpod-io/gitpod/blob/c43a6c9859754a06c425d8ec490d54e9978a8a6a/components/server/src/workspace/workspace-starter.ts#L1390) if I'm not mistaken